From 76f7c7e6efd8166342cc3c84e53d4e18dbdeda13 Mon Sep 17 00:00:00 2001
From: Mingli Yu <mingli.yu@windriver.com>
Date: Thu, 28 Jan 2021 18:44:10 -0800
Subject: [PATCH] setup_build.py: avoid absolute path

Prevent absolute path from ending up in the egg-info SOURCES.txt.

Upstream-Status: Submitted [https://github.com/h5py/h5py/pull/1803]

Signed-off-by: Mingli Yu <mingli.yu@windriver.com>
Signed-off-by: Leon Anavi <leon.anavi@konsulko.com>
---
 setup_build.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup_build.py b/setup_build.py
index 9b4766e..a411462 100644
--- a/setup_build.py
+++ b/setup_build.py
@@ -21,7 +21,7 @@ from setup_configure import BuildConfig
 
 
 def localpath(*args):
-    return op.abspath(op.join(op.dirname(__file__), *args))
+    return op.join(*args)
 
 
 MODULES_NUMPY2 = ['_npystrings']
